<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Instrument:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="instrument_comboBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Post Processing</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QRadioButton" name="mantid_post_processing_button">
           <property name="text">
            <string>Mantid</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="idl_post_processing_button">
           <property name="text">
            <string>IDL</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cache Dir.:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cache_dir_button">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cache_dir_label">
        <property name="text">
         <string>N/A</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Output Dir.:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="output_dir_button">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="output_dir_label">
        <property name="text">
         <string>N/A</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="idl_groupbox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Browse ...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>idl_post_processing_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>post_processing_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mantid_post_processing_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>post_processing_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>instrument_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>instrument_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cache_dir_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cache_dir_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>output_dir_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>output_dir_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>post_processing_clicked()</slot>
  <slot>instrument_changed()</slot>
  <slot>cache_dir_button_clicked()</slot>
  <slot>output_dir_button_clicked()</slot>
  <slot>add_key_value()</slot>
  <slot>remove_key_value_selected()</slot>
 </slots>
</ui>
